neuro_san/test/interfaces/null_assert_forwarder.py

```diff
@@ -0,0 +1,217 @@
+# Copyright (C) 2023-2025 Cognizant Digital Business, Evolutionary AI.
+# All Rights Reserved.
+# Issued under the Academic Public License.
+#
+# You can be released from the terms, and requirements of the Academic Public
+# License by purchasing a commercial license.
+# Purchase of a commercial license is mandatory for any use of the
+# neuro-san SDK Software in commercial settings.
+#
+# END COPYRIGHT
+from typing import Any
+
+from neuro_san.test.interfaces.assert_forwarder import AssertForwarder
+
+
+class NullAssertForwarder(AssertForwarder):
+    """
+    Implementation of the AssertForwarder interface that does nothing for each assert type.
+    This is useful as a superclass for other AssertForwarders that only want to focus
+    on a few specific asserts.
+    """
```

Comment:
Implementation of AssertForwarder that specifically does nothing so that subclasses can focus on their logic and not what they don't want to collect.